code/modules/projectiles/guns/energy/pulse.dm

```diff
@@ -24,11 +24,11 @@
 /obj/item/weapon/gun/energy/pulse_rifle/destroyer
 	name = "pulse destroyer"
 	desc = "A heavy-duty, pulse-based energy weapon. Because of its complexity and cost, it is rarely seen in use except by specialists."
-	cell_type = /obj/item/weapon/cell/super
-	fire_delay = 25
+//	cell_type = /obj/item/weapon/cell/super
+	fire_delay = null		//Aspawn only gun doesn't really need to be balanced against the station/antags

```

Comment:
From my memory, asking the Polaris staff about this weapon (and the SAW) and the initial change was met with roughly "Who cares, it's never spawned". I can't say I've ever seen this thing spawned on Polaris. I'm highly dubious that it will ever be made accessible in regular situations. If a downstream decides to give the fire delay back, it's not exactly difficult.